specification/common-types/resource-management/v2/types.json

```diff
@@ -638,6 +638,16 @@
     }
   },
   "parameters": {
+    "NotificationIdParameter": {
```

Comment:
the types.json is shared across all RPs, and this is not a good place if you just add one parameter that is needed by your RP. 
btw, what's NotificationIdParameter meaning?

Reply:
Hi thanks for the review. I will remove this it from this file as I only use it in one API I touch.